src/Main.java

```diff
@@ -0,0 +1,32 @@
+import box.task.ThingsProcessingDemonstrator;
+
+import functionalinterface.example.FunctionalInterfacesDemonstrator;
+import movie.task.MovieProcessingDemonstrator;
+import users.task.UserProcessingDemonstrator;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter number of task:(1-4)");
+        switch (sc.nextInt()){
+            case 1:
```

Comment:
`1, 2, 3, 4` --> move to constants

Reply:
In my opinion is`s comlete nonsence, but, done.